hsl/router-config.json

```diff
@@ -1,9 +1,8 @@
 {
   "routingDefaults": {
-    "searchWindow": "1h",
+    "maxSearchWindow" : "14h",
```

Comment:
I think this is configured on the wrong level. It should be inside the `transit` block.